packages/devtools_app/lib/src/memory/flutter/memory_screen.dart

```diff
@@ -255,23 +253,34 @@ class MemoryBodyState extends State<MemoryBody> with AutoDisposeMixin {
       child: Row(
         crossAxisAlignment: CrossAxisAlignment.center,
         children: [
-          OutlineButton(
-            key: MemoryScreen.pauseButtonKey,
-            onPressed: controller.paused ? null : _pauseLiveTimeline,
-            child: Label(
-              pauseIcon,
-              'Pause',
-              minIncludeTextWidth: _primaryControlsMinVerboseWidth,
-            ),
+          ValueListenableBuilder(
+            valueListenable: controller.paused,
+            builder: (context, paused, _) {
+              return OutlineButton(
+                key: MemoryScreen.pauseButtonKey,
+                onPressed: paused ? null : controller.pauseLiveFeed,
+                child: Label(
+                  pauseIcon,
+                  'Pause',
+                  minIncludeTextWidth: _primaryControlsMinVerboseWidth,
+                ),
+              );
+            },
           ),
-          OutlineButton(
-            key: MemoryScreen.resumeButtonKey,
-            onPressed: controller.paused ? _resumeLiveTimeline : null,
-            child: Label(
-              playIcon,
-              'Resume',
-              minIncludeTextWidth: _primaryControlsMinVerboseWidth,
-            ),
+          const SizedBox(width: denseSpacing),
+          ValueListenableBuilder(
+            valueListenable: controller.paused,
```

Comment:
why not wrap the whole row in a the ValueListenableBuilder instead of wrapping both buttons in the same one?

Reply:
done -